frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -394,7 +404,11 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                         <LemonDivider />
                         <FeatureFlagRollout />
                         <LemonDivider />
-                        <FeatureFlagReleaseConditions />
+                        <FeatureFlagReleaseConditions
+                            id={`${featureFlag.id}`}
```

Comment:
I'm pretty sure this would produce `"null"`

```suggestion
                            id={featureFlag.id ? String(featureFlag.id) : undefined}
```

Reply:
which is ok, sharing ids across all new flags (there can be only one at a time) 😅